cmd/livepeer_cli/wizard_eth.go

```diff
@@ -16,3 +19,25 @@ func (w *wizard) setGasPrice() {
 
 	httpPostWithParams(fmt.Sprintf("http://%v:%v/setGasPrice", w.host, w.httpPort), val)
 }
+
+func (w *wizard) signMessage() {
+	fmt.Printf("Enter or paste the message to sign, then press ctrl + d:\n")
+	mystr := []string{""}
+	text, err := w.in.ReadString('\n')
```

Comment:
Minor: If we declare `text` and `err` before the loop, we could avoid the extra `ReadString()` call outside of the loop. Something like:

```
var text string
var err error
for err != io.EOF {
    if text, err = w.in.ReadString('\n'); err != io.EOF {
        ...
    }
}
```